drivers/hmis/app/graphql/types/hmis_schema/client_sort_option.rb

```diff
@@ -0,0 +1,18 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::ClientSortOption < Types::BaseEnum
+    description 'HUD Client Sorting Options'
+    graphql_name 'ClientSortOption'
+
+    value 'LAST_NAME', 'Client Last Name', value: 'LastName'
```

Comment:
Should e define this as `LAST_NAME_DESC` or does the asc/desc get set somewhere else?